File: WebApi/WebAPI/Models/Tables/Province.cs
namespace WebAPI.Models.Tables
{

    public class Province
    {
        public string ProvinceId { get; set; }

        public string RegionId { get; set; }

        public string EnName { get; set; }

        public string DrName { get; set; }

        public string PaName { get; set; }

        public bool IsActive { get; set; }
    }
}

File: HAP-SPA/src/app/models/Province.ts
export interface Province{
    provinceId: string;
    regionId: string;
    regionName: string;
    enName: string;
    drName: string;
    paName: string;
    isActive?: boolean;
    }
export interface Region{
    regionID: string;
    enName: string;
    drName: string;
    paName: string;
    isActive?: boolean;
    }